src/StatusQ/Controls/StatusCheckBox.qml

```diff
@@ -8,16 +8,20 @@ import StatusQ.Components 0.1
 
 
 CheckBox {
-    id: statusCheckBox
+    id: root
+
+    property bool leftSideCheckbox: true
```

Comment:
I suggest to change the name as it is indeed a property of the `CheckBox` component. It seems redundant for me. WDYT?

```suggestion
    property bool leftSide: true
```